mathesar/errors.py

```diff
@@ -1,2 +1,53 @@
+from collections import namedtuple
+from contextlib import contextmanager
+
+from rest_framework.exceptions import APIException
+
+
 class InvalidTableError(Exception):
     pass
+
+
+ExceptionTransformerDetail = namedtuple(
```

Comment:
@silentninja I updated the whitespace here to make it more easily readable (in my opinion) and to match the rest of the code style. I figured updating it directly would be easier.